.gitignore

```diff
@@ -12,3 +12,8 @@ version.h
 
 config.mk
 config.h
+
+description-pak
+doc-pak/
+*.deb
+
```

Comment:
I don't think it makes sense to add gitignore entries for artifacts generated by other software that's not the build system.

Reply:
Understood, removed with today's commit.